src/SamplesApp/UITests.Shared/ItemExclusions.props

```diff
@@ -17,10 +17,6 @@
 		<Compile Remove="$(MSBuildThisFileDirectory)Microsoft_UI_Xaml_Controls\PagerControlTests\**" />
 		<None Include="$(MSBuildThisFileDirectory)Microsoft_UI_Xaml_Controls\PagerControlTests\**" />
 
-		<Page Remove="$(MSBuildThisFileDirectory)Windows_UI_Xaml_Controls\Repeater\**" />
-		<Compile Remove="$(MSBuildThisFileDirectory)Windows_UI_Xaml_Controls\Repeater\**" />
-		<None Include="$(MSBuildThisFileDirectory)Windows_UI_Xaml_Controls\Repeater\**" />
-
```

Comment:
Tried that before and didn't work.

These will need to move to the below `ItemGroup` with (the one with `Condition="'$(UseWinUI)'=='true'"`)

Even then, `FlowLayout_Simple` sample isn't going to compile on WinAppSDK specifically, so that sample will need to be excluded for WinAppSDK builds